lib/pwwka/transmitter.rb

```diff
@@ -1,3 +1,5 @@
+require 'resque'
```

Comment:
Is resque in the gemspec?  I guess either it should be or we should do something to handle if it's not there so this won't blow up for people that dont' need/want/can't have resque.